src/mme/mme-gn-handler.h

```diff
@@ -44,8 +44,9 @@ void mme_gn_handle_sgsn_context_acknowledge(
 void mme_gn_handle_ran_information_relay(
         ogs_gtp_xact_t *xact, ogs_gtp1_ran_information_relay_t *req);
 
-void guti_to_rai_ptmsi(const ogs_nas_eps_guti_t *nas_guti, ogs_nas_rai_t *rai,
-        mme_p_tmsi_t *ptmsi, uint32_t *ptmsi_sig);
+void rai_ptmsi_to_guti(const ogs_nas_rai_t *rai, mme_p_tmsi_t ptmsi, uint32_t ptmsi_sig, ogs_nas_eps_guti_t *nas_guti);
```

Comment:
why is this added?

Reply:
This is the error I got without:
```
../src/mme/mme-gn-handler.c:83:6: error: no previous prototype for ‘rai_ptmsi_to_guti’ [-Werror=missing-prototypes]                             
   83 | void rai_ptmsi_to_guti(const ogs_nas_rai_t *rai, mme_p_tmsi_t ptmsi, uint32_t ptmsi_sig, ogs_nas_eps_guti_t *nas_guti)                  
      |      ^~~~~~~~~~~~~~~~~                                                                                                                  
cc1: all warnings being treated as errors   
```

Reply:
Never mind, somehow I didn't see that it was originally static. Removed